### PR TITLE
Add source/destination input fields in the navbar

### PR DESCRIPTION
**Pull request by michaelmartinelli15**

### CODE REVIEW

File: src/components/Directions.tsx [hunk 1]
```diff
@@ -64,6 +64,8 @@ const buildAndDisplayRoute = (
                     });
                 });
                 selectCameras(response);
+            } else if (status == "NOT_FOUND") {
```
Comment (kylebrain) [inline src/components/Directions.tsx:67]: Really should add a submit button, it alerts me that I've hit a QUERY_LIMIT

**Review by kylebrain — CHANGES_REQUESTED** [review]

File: src/components/MapScreen.tsx [hunk 2]
```diff
@@ -20,8 +27,8 @@ export const MapScreen: React.FC<MapScreenProps> = ({ selectCameras, webcams }:
             }}
         >
             <Directions
-                source="Reno, NV"
-                destination="Incline Village, NV"
+                source={source}
```
Comment (kylebrain) [inline src/components/MapScreen.tsx:30]: Need handle route fetching in App as well, but this can be in another patch. Currently only updates the webcams if in the map screen.

File: src/components/Directions.tsx (hunk 1 above)
Comment (kylebrain) [reply src/components/Directions.tsx:67]: I get it now, you set the field in the dashboard and then switch to the maps. I'll approve for now, but we'll need some kind of submit button for the form or refresh button on the maps screen in a future patch.

**Review by kylebrain — APPROVED** [review]

**Review by GarrettMoore22 — APPROVED** [review]
Looks good!
